config/dynamicconfig/development_es.yaml

```diff
@@ -26,15 +26,8 @@ frontend.validSearchAttributes:
       CustomDatetimeField: "Datetime"
       TemporalChangeVersion: "Keyword"
       BinaryChecksums: "Keyword"
-      project: "Keyword"
-      service: "Keyword"
-      environment: "Keyword"
-      addon: "Keyword"
-      addon-type: "Keyword"
-      user: "Keyword"
       CustomNamespace: "Keyword"
       Operator: "Keyword"
-      RolloutId: "Keyword"
-      
```

Comment:
why are these attr removed?

Reply:
They are not used and looks like some uber/uDeploy legacy. Do you have better context on it? This is not directly related to this change. I think I would better create another PR for them.